### ISSUE TITLE
[BUG] MediaElement 4.0.0 Volume control

### ISSUE THREAD

**Issue by Pmr-precure** [CommunityToolkit/Maui#1988]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Did you read the "Reporting a bug" section on Contributing file?

- [X] I have read the "Reporting a bug" section on Contributing file: https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#reporting-a-bug

### Current Behavior

In version 4.0.0 (Android) when showing a video, while having a FOREGROUND_SERVICE running, the volume bar will show. See screenshot:

![shared image](https://github.com/CommunityToolkit/Maui/assets/99671597/d367bbd3-9b8b-47c6-af6e-b182d61422d7)

This does not happen in version 3.1.1

### Expected Behavior

Not showing volume bar before playing a video, while having a foreground service runnning.

### Steps To Reproduce

Clone repo.

Run the project observe Volume bar when the video starts playing.
(if it does not happen rerun the project)

Downgrade to version 3.1.1, observe it does not happen.

### Link to public reproduction project repository

https://github.com/Pmr-precure/test_repo/tree/main

### Environment

```markdown
- .NET MAUI CommunityToolkit: 9.0.1
- OS: Android
- .NET MAUI: 8
```

### Anything else?

Bare in mind, this only happens if you have a Foreground service running.

**Comment by ne0rrmatrix** [comment]
@vhugogarcia can u close this please? It has been resolved via #1965

**Comment by Pmr-precure** [comment]
> @vhugogarcia can u close this please? It has been resolved via #1965

In what version? Because this happens to me in 4.0.0

**Comment by vhugogarcia** [comment]
Thanks @ne0rrmatrix and @Pmr-precure for the update.

**Comment by Pmr-precure** [comment]
> Thanks @ne0rrmatrix and @Pmr-precure for the update.
> 
> Edit: Sorry, I clicked the close button instead of comment button, hehehe!.
> 
> I will close this issue once the new version is released with the fix. I know the fix is merged, but no new version has been released. So, I will keep an eye on it.

Thanks!

**Comment by Pmr-precure** [comment]
Fixed in 4.0.1
